docs/node/tutorial.mdx

```diff
@@ -151,6 +152,58 @@ Current validator set:
   1. {pubkey = a2d8e307117695c8d0d3adbda136cb2ad630c43b8672a0eba46d79501831b794, power = 1}
 ```
 
+## Deploying A Database
+
+To deploy a database to the network, we will need to locate the network chain ID and any of the node's HTTP listen addresses.
+
+### Getting The Chain ID
+
+Use the `kwil-admin node status` command from earlier to get the chain ID:
+
+```bash
+$ kwil-admin node status --rpcserver unix:///tmp/node0.sock
+{
+  "node": {
+    # highlight-next-line
+    "chain_id": "kwil-chain-WfnGxpWd", # chain ID
+    "name": "MacBook-Pro-7.local",
+    ...
+  },
+```
+
+### Getting The Node's HTTP Listen Address
+
+A node's HTTP listen address can be found in its `config.toml` file. To display the config information, use the `kwil-admin node dump-config` command:
```

Comment:
This is a good example of the `node dump-config` command I suppose, but also something that I'd personally never do to figure out the RPC endpoint.  It's a decent example, so might as well go with it, but for reference, I'd do one of: (a) check my config.toml on disk, (b) look at the `kwild` logs on start up, or (c) use netstat or similar to see what hosts+ports `kwild` is really listening on.

Reply:
Fair on using other tools to check where it is listening, but remember, we need this to be NOOB PROOF. Jon, you are an S tier hacker. I am a simpleton. I do not have your powers.

But actually, one of the pieces of feedback we got was that looking at the config.toml, it isn't really clear which address to point the CLI to in order to deploy the database. For example, there is `app.http_listen_addr`, `app.grpc_listen_addr`, `app.admin_listen_addr`, `chain.rpc.listen_addr`, `chain.p2p.listen_addr`, and persistent peers if configured. Which of those addresses should be given to kwil-cli configure? It isn't clear unless you understand a bit about how Kwil works and the purposes of each of these addresses. So i am trying to figure out the simplest way for someone to use this tutorial and get the http address for their node.

Reply:
We could tell them to look in the config.toml and show where to look, but again, I think there is value having there be very tight guardrails/training wheels so people don't somehow veer off course.